chapters/ch50.md

```diff
@@ -1,35 +1,35 @@
 > # 50. Choosing pipeline components: Data availability
 
---> _replace THIS LINE by your translation for the above line_
+# 50. Lựa chọn các thành phần cho pipeline: Dữ liệu sẵn có
```

Comment:
"data  availability" và "availability of data" có giống nhau ko bác nhỉ?

Reply:
ừ mình cũng nghĩ "Data availability" -> "Tính sẵn có của dữ liệu" chính xác hơn.
Dữ liệu sẵn có == Available data.

Reply:
Okie, mình sẽ đổi lại thành "tính sẵn có của dữ liệu".